### PR TITLE
Fix detection for XP-Pen devices with PID: 117

### LINKED ISSUE #698: G430/S and G540
## Description

It seems that G430, G430S (rev A?), and G540 have the same PID and VID.

We should probably try to determine which is which through some other methods like device strings.

## Related Issues

#662

### PR DESCRIPTION
**Pull request by X9VoiD**
## Changes

- Brought back G430S v2
- Use device string index 2 for identification

| Tablet | Identifier | "LPI" |
| --- | --- | :---: |
| G430 | `"TABLET G3 4x3"` | n/a |
| G430S | `"G430S"` | 5080 |
| G540 | `"TABLET G3 5x4"` | 5080 |

## Verification

- [x] G430
- [x] G430S
- [x] G430S v2
- [x] G540

Fixes #698

### CODE REVIEW

**Comment by InfinityGhost** [comment]
Possibly already resolved in a different PR, a re-review is probably worth doing.

**Comment by X9VoiD** [comment]
Did a re-review and this PR is still needed. Fixed merge conflict.

File: OpenTabletDriver/Configurations/XP-Pen/Star G430.json [hunk 1]
```diff
@@ -21,31 +21,9 @@
       "FeatureInitReport": null,
       "OutputInitReport": "ArAC",
       "DeviceStrings": {
-        "4": "V2\\.0 20170423"
```
[the remark this answers is not in the document]
Comment (X9VoiD) [reply OpenTabletDriver/Configurations/XP-Pen/Star G430.json:24]: It's our only way of determining if it's a G430 or G540.

**Comment by Kyuunex** [comment]
I think this PR sets incorrect physical dimensions for G430
"Width": 114.3,
"Height": 73.025,
above seem incorrect

these are correct
"Width": 101.6,
"Height": 76.2,

**Comment by X9VoiD** [comment]
G430 have an LPI of 10160 or LPmm of 400, which means
```
45720 / 400 = 114.3
29210 / 400 = 73.025
```

**Comment by Kyuunex** [comment]
I physically measured my G430 with a ruler to get these values though

**Comment by X9VoiD** [comment]
Did you verify your measurements by actually testing if the pen tracks on it? Because if you say that that measurements are correct then it means that G430 have a non 1:1 sensor aspect ratio.

**Comment by X9VoiD** [comment]
Can't find anything wrong with the measurements so will just say that G430 is an exemption and allow non 1:1 sensor ratio.

**Pull request merged**